src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/RichTextEditorTextArea.kt

```diff
@@ -0,0 +1,59 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.Patience
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.openqa.selenium.By
+import org.openqa.selenium.JavascriptExecutor
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.openqa.selenium.support.ui.ExpectedConditions.*
+import java.time.Duration
+
+/**
+ * Helper class for Rich Text Editor enabled Textarea
+ *
+ */
+class RichTextEditorTextArea(
+    private val driver: WebDriver
+) {
+
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    private fun clearAndTypeIfPresent(id :String, text: String) {
+        if (driver.isElementPresent(By.id(id))) {
+            val webElement = driver.findElement(By.id(id))
+            driver.wait(timeout = Duration.ofSeconds(2), condition = elementToBeClickable(webElement))
+            webElement.clear()
+            webElement.sendKeys(text)
+        }
+    }
+
+    fun clearAndTypeIfPresentRte(id : String, text: String) {
+        //check richeditor-cover exists in class attribute
+        if (driver.isElementPresent(By.id(id))) {
```

Comment:
I found that `EditIssuePage` does that:
```kotlin
summaryLocator.clearAndTypeIfPresent("summary")
descriptionLocator.clearAndTypeIfPresent("description")
form.fillRequiredFields()
```
However, it makes sense there, because:
* summary and description might actually be absent, we've got a customer with such setup
* we at least the required fields, ensuring the edit dialog is submittable

Reply:
oh, got it, will do